frontend/src/components/DiscoverStreamerCard/DiscoverStreamerCard.tsx

```diff
@@ -22,12 +23,18 @@ export default function DiscoverStreamerCard({
   isConnectionsPage,
   onStreamerClick,
   onStreamerConnectClick,
+  onStreamerNotInterestedClick
```

Comment:
Ditto before, this component should be abstracted from the business logic and just render left or right instead of specific actions depending on the context